errors/default/report.phtml

```diff
@@ -17,6 +17,10 @@
  * @copyright   Copyright (c) 2006-2020 Magento, Inc. (http://www.magento.com)
  * @license     http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
  */
+
+/**
+ * @var Error_Processor $this
```

Comment:
Can we maybe make this 1 line instead of the 3? Not sure if it would look better. Not the most important thing obviously